guide/foundations/principles.md

```diff
@@ -172,11 +207,17 @@ The Bitcoin network doesn’t need to know your name for you to use it. Strive t
 
 Unlike traditional banking systems, the Bitcoin economy does not require new users to seek permission from anyone. Bitcoin has no central point of control. No one person or entity is in charge. Connecting to any [node]({{ '/guide/glossary/#node' | relative_url }}) on the network gives you the same rights and responsibilities, ensuring no single point of failure.
 
-**Do**
+{% include tip/open.html color="green" icon="check" label="Do this" %}
+
 - Design products that encourage people to run a full Bitcoin node
-- Alternatively, use a light client with the [p2p network]({{ 'https://github.com/bitcoin/bips/blob/master/bip-0157.mediawiki'}}) using [compact block filters]({{ 'https://github.com/bitcoin/bips/blob/master/bip-0158.mediawiki'}})
+- Alternatively, use a light client with the p2p network using compact block filters
```

Comment:
Why remove the links?

Reply:
I copied the text from the site - long story, was not intentional, will amend today :)